test/cpp/interop/client.cc

```diff
@@ -57,6 +57,7 @@ DEFINE_string(
     "half_duplex : half-duplex streaming;\n"
     "jwt_token_creds: large_unary with JWT token auth;\n"
     "large_unary : single request and (large) response;\n"
+    "long_connection: sends large_unary rpcs over a single long connection;\n"
```

Comment:
I might actually rename this from `long_connection` to `long_lived_channel`, because `long_connection` might be confusing: not all of the RPC's made on the channel are guaranteed to be on the same connection